src/components/Dashboard/PrintDashboard.js

```diff
@@ -63,7 +78,10 @@ export class PrintDashboard extends Component {
                 })
             }
 
-            this.props.addDashboardItem({ type: PRINT_TITLE_PAGE })
+            this.props.addDashboardItem({
+                type: PRINT_TITLE_PAGE,
+                isOipp: true,
```

Comment:
What does `Oipp` stand for?

Reply:
one-item-per-page. It isn't really described anywhere though, good point.

Reply:
prop renamed